libs/libRocket/Source/Core/precompiled.cpp

```diff
@@ -27,3 +27,6 @@
 
 #include "precompiled.h"
 
+// XCode's ranlib complain on empty object, adding any function prevents this
+void RocketDummyFunction() {
```

Comment:
Either way, the commit needs to happen in the librocket repo so it doesn't get lost. then we update the librocket here by copying what's in the librocket repo.